icons/customink.svg

```diff
@@ -0,0 +1 @@
4.243-.785.34-.317.116-.654.214-1.373.395-.695.174-.837.213-1.214.327-.122.037-.289.085-.372.108-.868.237-1.562.755-1.924 1.436-.245.46-.35 1.095-.277 1.663a2.812 2.82 0 0 0 3.49 2.354 2.901 2.901 0 0 0 1.202-.66c.192-.177.443-.507.56-.74.08-.16.39-.871.485-1.111A13.51 13.51 0 0 0 9 15.595a7.93 7.93 0 0 0 .087-.918c.001-.156-.002-.177-.04-.255-.077-.158-.243-.214-.411-.14zm1.833-3.345c-.068.035-.1.096-.1.19.003.1.03.16.138.307a1.849 1.854 0 0 0 2.568.426 2.02 2.02 0 0 0 .533-.593c.064-.125.045-.254-.044-.315-.075-.05-.144-.044-.372.032a3.22 3.22 0 0 1-1.081.202 3.254 3.263 0 0 1-1.248-.187c-.252-.084-.33-.096-.394-.062zM20.87 9.716c-.557.077-.965.244-1.834.756-1.034.612-1.42.79-2.24 1.038-.141.042-.165.055-.238.126-.123.123-.14.22-.06.376.077.15.789.89 1.23 1.278.604.53 1.336 1.058 1.903 1.37l.296.163c.282.156.594.266.912.318.238.039.653.036.886-.007a2.681 2.689 0 0 0 1.455-.753c.437-.432.708-.963.798-1.57a3.596 3.605 0 0 0 0-.79 2.763 2.77 0 0 0-.738-1.508 2.574 2.58 0 0 0-1.369-.77c-.11-.025-.212-.03-.518-.035a5.5 5.515 0 0 0-.483.009zM2.348 9.712c-.462.07-.78.184-1.1.395a2.751 2.759 0 0 0-.993 3.47 2.732 2.74 0 0 0 3.36 1.452 5.61 5.61 0 0 0 .804-.39 12.02 12.051 0 0 0 2.925-2.42c.08-.09.159-.19.173-.22.082-.171.013-.345-.174-.44a1.26 1.264 0 0 0-.178-.063 8.102 8.124 0 0 1-.773-.27c-.409-.171-.745-.35-1.449-.768-1.024-.607-1.484-.76-2.258-.755a4.945 4.958 0 0 0-.337.009zM13.522 7.95a.998 1 0 0 0-.781.655 1.19 1.193 0 0 0-.023.533c.067.266.265.515.509.636.214.104.398.13.619.084a.96.963 0 0 0 .68-.533c.069-.144.074-.21.016-.258-.052-.046-.064-.046-.172.003a1.248 1.251 0 0 1-.168.06c-.158.042-.31-.028-.398-.18a.289.29 0 0 1-.045-.18c0-.23.146-.376.414-.417.121-.02.156-.126.068-.207a1.16 1.163 0 0 0-.37-.18 1.168 1.171 0 0 0-.349-.016zm-3.293 0a.991.994 0 0 0-.774.63 1.138 1.141 0 0 0-.03.557c.067.267.267.517.51.637a.943.946 0 0 0 1.012-.094c.167-.125.32-.358.335-.51.006-.067.002-.077-.035-.106-.023-.019-.05-.034-.06-.034a.96.96 0 0 0-.14.05c-.147.064-.212.075-.305.05a.372.373 0 0 1-.232-.182.286.286 0 0 1-.044-.176c0-.23.145-.378.414-.419.104-.017.15-.116.092-.189a.963.963 0 0 0-.324-.174 1.035 1.038 0 0 0-.419-.04zM11.467.004c-.35.02-.634.04-.872.068-2.27.249-4.318 1.097-6.062 2.503a13.23 13.23 0 0 0-1.425 1.366A12.691 12.725 0 0 0 .79 7.79c-.127.33-.126.485.005.631a.41.41 0 0 0 .325.129c.102-.002.134-.01.303-.08.368-.15.62-.195 1.024-.18.406.013.695.073 1.609.337 1.507.435 2.465.576 3.194.47.289-.041.586-.134.647-.2.066-.072.062-.127-.038-.424-.184-.55-.26-.98-.26-1.48a4.366 4.377 0 0 1 3.644-4.307 4.661 4.661 0 0 1 1.37-.015 4.395 4.407 0 0 1 3.064 1.947 4.348 4.36 0 0 1 .419 3.934c-.103.27-.078.349.137.428.322.12.872.18 1.312.143.638-.053 1.415-.214 2.383-.496.888-.259 1.203-.326 1.616-.338.4-.012.676.036 1.014.18.079.034.182.069.23.078a.438.44 0 0 0 .449-.193c.04-.068.045-.086.043-.205 0-.12-.006-.146-.08-.336a13.667 13.703 0 0 0-1.184-2.35 11.942 11.974 0 0 0-3.397-3.499C16.93.831 15.028.186 12.893.023a25.233 25.3 0 0 0-1.425-.02Z"/></svg>
```

Comment:
When overlapped with the original, I see some [some differences!](https://github.com/simple-icons/simple-icons/assets/35000807/bc8d2240-e70a-4e75-adea-e2467bdfbef9)

Can you please re-do? Edit the icon at maximum precision before running through SVGOMG

Reply:
What is the correct precision it is showing at your end?

Reply:
I can see it is 3 from the code itself. To check, I myself scaled down original and ran it through SVGO at precision 3 and the resulting icon was matching well with original.
Remember to scale proportionally and work with maximum available precision on your editor app.

Reply:
@PratyushJayachandran I have updated the image, can you review it now.

Reply:
@patilanuja unfortunately the issue still remains :(
I am attaching a comparison of [your contribution with original](https://github.com/simple-icons/simple-icons/assets/35000807/bb8c5698-6f15-411c-b00d-5c8168221e6a) and [my version compared to original](https://github.com/simple-icons/simple-icons/assets/35000807/2709fc9e-c8fd-49ba-9bd8-4c195c172d71).

You may download them to see where it is going wrong. Since you are a first time contributor, we would like you to understand the issue and try again.

I am not sure what your program settings are but it could be a precision issue with the program itself where it is rounding of the dimensions when scaling the image.

Reply:
@PratyushJayachandran I fixed it, can you check it again?

Reply:
Not perfect match but acceptable!

Reply:
Thank you for the contribution @patilanuja